src/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)..\obj\$(MSBuildProjectName)\</BaseIntermediateOutputPath>
-    <OutputPath>$(MSBuildThisFileDirectory)..\bin\$(MSBuildProjectName)\$(Configuration)\</OutputPath>
+    <BaseOutputPath Condition=" '$(BaseOutputPath)' == '' ">$(MSBuildThisFileDirectory)..\bin\$(MSBuildProjectName)\</BaseOutputPath>
```

Comment:
The sample in the docs at https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing-faq also declares OutputPath too:
```xml
<OutputPath Condition="'$(OutputPath)' == ''">$(BaseOutputPath)</OutputPath> 
```